model/model.go

```diff
@@ -68,6 +68,12 @@ type PluginMeta struct {
 	Extensions []string `json:"extensions" yaml:"extensions"`
 }
 
+type PluginFQN struct {
+	Registry string `json:"registry,omitempty" yaml:"registry,omitempty"`
```

Comment:
Can you elaborate why we need omitempty here?

Reply:
"Need" is perhaps strong, but I included `omitempty` because I would expect `unmarshal -> marshal` to be idempotent, and we don't require the `registry` field in `PluginFQN` (i.e. from Che, it could be null and so omitted).

Reply:
I see. Thanks for the explanation